packages/table/preview/index.tsx

```diff
@@ -479,18 +564,24 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
         }
     }
 
-    private updateBooleanState(stateKey: keyof IMutableTableState) {
+    private handleBooleanStateChange = (stateKey: keyof IMutableTableState) => {
         return handleBooleanChange((value: boolean) => {
             this.setState({ [stateKey]: value });
         });
     }
 
-    private updateNumberState(stateKey: keyof IMutableTableState) {
+    private handleNumberStateChange = (stateKey: keyof IMutableTableState) => {
         return handleNumberChange((value: number) => {
             this.setState({ [stateKey]: value });
         });
     }
 
+    private handleStringStateChange = (stateKey: keyof IMutableTableState) => {
+        return handleStringChange((value: string) => {
```

Comment:
def don't need the typing on `value` param in all three cases here, `handle*Change` does that for you.

then you could make these bodies into one-liners:

```ts
private handleNumberStateChange = (stateKey: keyof IMutableTableState) => {
    return handleNumberChange((value) => this.setState({ [stateKey]: value }));
}
```